src/navigation/SideBar/SideBar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
```

Comment:
These changes aren't expected in this PR, isn't it?

Reply:
Removed